Makefile

```diff
@@ -652,7 +651,7 @@ $(SBOM): $(ROOTFS_TAR)| $(INSTALLER)
 
 $(GOSOURCES):
```

Comment:
Shouldn't you be changing the COMPARESOURCES target (instead of GOSOURCES)?

Reply:
@eriknordmark fixed now. Had multiple repo's checked out for eve so fixed in the wrong one and pushed the wrong one